clients/py/sapphirepy/deoxysii.py

```diff
@@ -0,0 +1,580 @@
+"""
+Copyright (c) 2019 Oasis Labs Inc. <info@oasislabs.com>
```

Comment:
I see the reference [here](https://github.com/oasisprotocol/deoxysii/blob/2091330c22b71dd4e04e15306498e68f3a1b3afc/impl.go#L1), but perhaps this should be OPF?

Reply:
Eh, it's a direct port of the Go client, so technically a derivative work *shrugs*